src/domain/space/space/space.entity.ts

```diff
@@ -102,6 +102,10 @@ export class Space extends AuthorizableEntity implements ISpace {
   @Column('char', { length: UUID_LENGTH, nullable: true })
   levelZeroSpaceID!: string;
 
+  @ManyToOne(() => Space)
+  @JoinColumn({ name: 'levelZeroSpaceID' })
+  levelZeroSpace?: Space;
```

Comment:
This may create some sort of circular dependency, since a root Space level zero Space is itself.
I would not dismiss it lightly...

Reply:
Indeed this one may well cause issues - I explicitly avoided it when doing the original implementation. Please call after the daily to discuss.

Reply:
@hero101 can you, please take another look?